source/_posts/2023-10-04-release-202310.markdown

```diff
@@ -367,6 +367,17 @@ backup was initiated. Instead, the local date and time are used for the name.
 
 {% enddetails %}
 
+{% details "Twitch" %}
```

Comment:
This part is already merged in another PR. Please remove the release notes adjustments from this PR.